spec/lib/pre_assembly/batch_spec.rb

```diff
@@ -11,6 +11,11 @@
     Cocina::RSpec::Factories.build(:dro, type: Cocina::Models::ObjectType.image).new(access: dro_access)
   end
   let(:dor_services_client_object) { instance_double(Dor::Services::Client::Object, find: item, update: true) }
+  let(:relative_file_paths) do
+    batch.digital_objects.map do |digital_object|
+      digital_object.object_files.map(&:relative_path)
+    end
+  end
 
```

Comment:
so we can verify that the relative_path on the individual files is set correctly in the digital objects